packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -139,6 +141,18 @@ export const checkTeamsLimit = async (orgId: string, dataLoader: DataLoaderWorke
 
   if (!(await isLimitExceeded(orgId, dataLoader))) return
 
+  if (!organization.activeDomain) return
```

Comment:
@tghanken We have few things that we can run only if organization has an active domain, including querying HubSpot API.
So I completely disabled the limits check for organizations without domain, is it okay?

Reply:
+1 I think this should be a bit more explicit discussion. While there are not many teams which qualify with no `activeDomain`, I don't see why we should exclude them from the teams limit.